connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
Sorry for the late response and thanks for investigating @kushalagrawal1. Too bad it's not done just with resteasy.
Since you already updated resteasy on your system, do you mind adding another commit with that?